SoObjects/SOGo/SOGoGCSFolder.m

```diff
@@ -1637,17 +1637,14 @@ - (NSArray *) _aclsFromGroupRoles: (NSArray *) records
       currentUID = [record valueForKey: @"c_uid"];
       if ([currentUID hasPrefix: @"@"])
         {
-	  group = [[SOGoCache sharedCache] groupNamed: currentUID  inDomain: domain];
-
-	  if (!group)
-	    {
-	      group = [SOGoGroup groupWithIdentifier: currentUID
-				 inDomain: domain];
-	      [[SOGoCache sharedCache] registerGroup: group  withName: currentUID  inDomain: domain];
-	    }
-
-          if (group && [group hasMemberWithUID: uid])
-            [acls addObject: [record valueForKey: @"c_role"]];
+          NSString *dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: currentUID];
```

Comment:
Respect the domain by using `[SOGoUserManager contactInfosForUserWithUIDorEmail:inDomain:]`.

Reply:
Resolved in commit 948c89f